lib/sm/client.rb

```diff
@@ -110,6 +111,8 @@ def connection
         conn.use :breakers
         conn.request :multipart
         conn.request :json
+        # conn.request :curl, ::Logger.new(STDOUT), :warn
```

Comment:
Why not Rails.logger? If you're going to have it commented out, should we just hold off on this PR? Should we include it in other places?